app/templates/administrative-units/administrative-unit/governing-bodies/governing-body.hbs

```diff
@@ -1,9 +1,9 @@
-{{page-title this.model.adminGovModel.governingBody.classification.label}}
+{{page-title this.model.governingBodyClassification.label}}
 
 {{breadcrumb
-  this.model.adminGovModel.governingBody.classification.label
-  route="administrative-units.administrative-unit.governing-bodies"
-  model=(array this.model.administrativeUnit.id this.model.governingBody.id)
+  this.model.governingBodyClassification.label
+  route="administrative-units.administrative-unit.governing-bodies.governing-body"
+  models=(array this.model.administrativeUnit.id this.model.governingBody.id)
```

Comment:
This was the actual issue:
- model vs models
- the wrong id was being passed (top level governing body instead of the "time specialisation one")